src/pip/_internal/utils/misc.py

```diff
@@ -20,6 +20,7 @@
 from pip._vendor import pkg_resources
 # NOTE: retrying is not annotated in typeshed as on 2017-07-17, which is
```

Comment:
We should keep this comment above the `retry` import.

Reply:
Fixed in https://github.com/pypa/pip/pull/8054/commits/6c1030ca95f22e6111d1307e6cf9a032d7a063e7 .